### PR TITLE
Display time link is scheduled for beside link status

### PR DESCRIPTION
**Pull request by Widdershin**

### CODE REVIEW

File: app/presenters/link_interaction_presenter.rb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "#{interaction.class.humanized_name} (#{status_and_scheduled_time})"
+  end
+
+  private
+
+  def status_and_scheduled_time
+    [
+      status,
+      scheduled_time_text
+    ].join(', ')
```
Comment (jakubgarfield) [inline app/presenters/link_interaction_presenter.rb]: compact before joining

```
irb(main):001:0> ["asd", nil].join(",")
=> "asd,"
irb(main):002:0> ["asd", nil].compact.join(",")
=> "asd"
```

File: app/presenters/link_interaction_presenter.rb [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+  end
+
+  def scheduled_time_text
```
Comment (jakubgarfield) [inline app/presenters/link_interaction_presenter.rb]: rename to something like `best_scheduled_time`

File: db/migrate/20150726055628_add_scheduled_time_to_link_interaction.rb [hunk 3]
```diff
@@ -0,0 +1,5 @@
+class AddScheduledTimeToLinkInteraction < ActiveRecord::Migration
+  def change
+    add_column :link_interactions, :scheduled_time, :datetime
```
Comment (jakubgarfield) [inline db/migrate/20150726055628_add_scheduled_time_to_link_interaction.rb]: as discussed, rename to something more relevant to _best_ time.

**Comment by Widdershin** [comment]
Fixed all that stuff @jakubgarfield :cat2:

**Comment by jakubgarfield** [comment]
#nailedit

**Pull request merged**